daffodil-core/src/main/scala/org/apache/daffodil/grammar/ElementBaseGrammarMixin.scala

```diff
@@ -642,7 +642,7 @@ trait ElementBaseGrammarMixin
 
   private lazy val textConverter = {
     primType match {
-      case _: NodeInfo.Numeric.Kind => ConvertTextNumberPrim(this)
+      case _: NodeInfo.Numeric.Kind => ConvertTextNumberStandardPrim(this)
```

Comment:
Yes, there are two kinds of textNumberRep, standard and zoned. 
The renaming is because this class is specifically about standard numbers, and the corresponding classes about zoned have "Zoned" in their names.

Reply:
I take your point that it should be ConvertTextStandardNumberPrim.